src/app/tests/suites/certification/Test_TC_DRLK_2_7.yaml

```diff
@@ -367,23 +343,24 @@ tests:
       response:
           error: INVALID_COMMAND
 
-    - label: "Clear a year day schedule for the first user"
-      PICS: DRLK.S.F0a && DRLK.S.C10.Rsp
-      command: "ClearYearDaySchedule"
+    - label: "Step 14:TH sends ClearUser Command to DUT with the UserIndex as 1"
+      PICS: DRLK.S.F08 && DRLK.S.C1d.Rsp
+      timedInteractionTimeoutMs: 10000
+      command: "ClearUser"
       arguments:
           values:
-              - name: "YearDayIndex"
-                value: 1
               - name: "UserIndex"
                 value: 1
 
-    - label: "Cleanup the created user with UserIndex 1"
-      command: "ClearUser"
-      timedInteractionTimeoutMs: 10000
+    - label: "Clear a year day schedule for the first user"
```

Comment:
I don't see this in the test plan.  Why is it here?

Reply:
@bzbarsky-apple Updated the step 14 and removed this step.